SlidingSegmentedControlTests/SlidingSegmentedControlTests.swift

```diff
@@ -82,6 +82,16 @@ class SlidingSegmentedControlTests: XCTestCase {
         XCTAssertEqual(slidingSegmentedControl.imageViews.map {$0.contentMode}, [.center, .center, .center])
     }
 
+    func testImageViewsIsAccessibilityElement() {
+        let slidingSegmentedControl = SlidingSegmentedControl(images: [UIImage(), UIImage()])
+        XCTAssertEqual(slidingSegmentedControl.imageViews.map { $0.isAccessibilityElement }, [true, true])
```

Comment:
@andris-zalitis 

Your claim about `EXC_BAD_INSTRUCTION` is **false**. 

We don't need that `guard` at all:

```swift
func testImageViewsImage() {
    let images = [UIImage(), UIImage()]
    let slidingSegmentedControl = SlidingSegmentedControl(images: images)
    XCTAssertEqual(slidingSegmentedControl.imageViews.first?.image, images[0])
    XCTAssertEqual(slidingSegmentedControl.imageViews.last?.image, images[1])
}
```

Tests with **no logic** are best.

P.S. Even better, one could simplify further:

```swift
// Do we have images views for all supplied images?
func testImageViewCount() {
    let images = [UIImage(), UIImage()]
    let slidingSegmentedControl = SlidingSegmentedControl(images: images)
    XCTAssertEqual(slidingSegmentedControl.imageViews.count, 2)
}

// Do we set proper images for image views?
func testImageViewsImages1() {
    let images = [UIImage(), UIImage()]
    let slidingSegmentedControl = SlidingSegmentedControl(images: images)
    XCTAssertEqual(slidingSegmentedControl.imageViews.first?.image, images[0])
}

...
```